Dockerfile

```diff
@@ -23,12 +23,12 @@ WORKDIR /layer8-app
 
 COPY --from=builder /build/main /layer8-app
 
-COPY --from=builder /build/.env /layer8-app
-
 COPY --from=builder /build/assets-v1 /layer8-app/assets-v1
 
 COPY --from=builder /build/certificates /layer8-app/certificates
 
+RUN touch /layer8-app/.env
```

Comment:
When we deploy the image to a service such as ECS/Kubernetes, we no longer use a .env file; instead, we store all configurations in the container environment variables. In this line of code, we add an empty .env file to ensure the application can still run locally, which is easier than putting the configuration in the environment variables.